Wikipedia/Code/PageHistoryFetcher.swift

```diff
@@ -180,24 +180,26 @@ extension HistoryFetchResults {
     fileprivate static func update(revisionsByDay: inout RevisionsByDay, revision: WMFPageHistoryRevision) {
         let distanceToToday = revision.daysFromToday()
         
-        if let existingRevisionsOnCurrentDay = revisionsByDay[distanceToToday] {
-            let sectionTitle = existingRevisionsOnCurrentDay.sectionTitle
-            let items = existingRevisionsOnCurrentDay.items + [revision]
-            revisionsByDay[distanceToToday] = PageHistorySection(sectionTitle: sectionTitle, items: items)
-        } else {
-            if let revisionDate = revision.revisionDate {
-                var title: String?
-                let getSectionTitle = {
-                    title = DateFormatter.wmf_long().string(from: revisionDate)
-                }
-                if Thread.isMainThread {
-                    getSectionTitle()
-                } else {
-                    DispatchQueue.main.sync(execute: getSectionTitle)
+        if (revision.user != nil) {        
```

Comment:
Consider using 'guard' here instead of 'if' - pretty handy way to keep the diff (and rightward indentation) minimal.